app/Actions/Database/StartDatabaseProxy.php

```
@@ -31,9 +31,35 @@ public function handle(StandaloneRedis|StandalonePostgresql|StandaloneMongodb|St
+                    if (! $containerName) {
+                        $containerName = "{$database->name}-" . generateApplicationContainerName($application);
+                    }
+                } else {
+                    $preview = $database->application_preview;
+                    $application = $preview?->application;
+                    $network = data_get($application, 'destination.network');
+                    $server = data_get($application, 'destination.server');
+                    $containerName = $preview && $application
+                        ? "{$database->name}-" . generateApplicationContainerName($application, $preview->pull_request_id)
+                        : null;
+                }
```

PHPMD (2.15.0)

[error] The method handle uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)